### ISSUE TITLE
官网好像没有Extentx这个模块了？

### ISSUE THREAD

**Issue by leemien** [yili1992/ExtentReport#1]
你好，我去官网找了下，发现没有Extentx的相关包了，是不是现在移除不维护了？

**Comment by yili1992** [comment]
是的,我那时候在用的时候 开发者正在 开发 名为Klov  java实现的一套server 来代替 extentx, 我的这个project 里面已经放了extentx

**Comment by yili1992** [comment]
> 你好，我去官网找了下，发现没有Extentx的相关包了，是不是现在移除不维护了？
是的,我那时候在用的时候 开发者正在 开发 名为Klov java实现的一套server 来代替 extentx, 我的这个project 里面已经放了extentx

**Comment by yili1992** [comment]
> 你好，我去官网找了下，发现没有Extentx的相关包了，是不是现在移除不维护了？

是的,我那时候在用的时候 开发者正在 开发 名为Klov java实现的一套server 来代替 extentx, 我的这个project 里面已经放了extentx

**Comment by leemien** [comment]
嗯，我也使用了一下这个，但是感觉这个页面展示效果没有之前的Extentx效果好，统计的数据也没有之前的好看
